{"uuid":"ef8e5472-395a-4ea5-96e4-28bed2a01627","historyId":"2c0336e29a8d4615631a93b851c62a09","fullName":"qa.fakestore.tests.AuthenticationTests.testValidLogin","labels":[{"name":"package","value":"qa.fakestore.tests.AuthenticationTests"},{"name":"testClass","value":"qa.fakestore.tests.AuthenticationTests"},{"name":"testMethod","value":"testValidLogin"},{"name":"parentSuite","value":"FakeStoreAPITestSuite"},{"name":"suite","value":"API Tests"},{"name":"subSuite","value":"qa.fakestore.tests.AuthenticationTests"},{"name":"host","value":"Burak"},{"name":"thread","value":"23852@Burak.TestNG-test=API Tests-2(37)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Valid User Login"},{"name":"epic","value":"Fake Store API Testing"},{"name":"feature","value":"Authentication"},{"name":"severity","value":"blocker"}],"links":[],"name":"testValidLogin","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Status code should be 200 expected [200] but found [201]","trace":"java.lang.AssertionError: Status code should be 200 expected [200] but found [201]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1419)\r\n\tat org.testng.Assert.assertEquals(Assert.java:1383)\r\n\tat qa.fakestore.tests.AuthenticationTests.testValidLogin(AuthenticationTests.java:29)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\r\n"},"stage":"finished","description":"Verify that user can login with valid credentials","steps":[],"attachments":[],"parameters":[],"start":1755374584958,"stop":1755374586812}